packages/lab/src/pill/Pill.css

```diff
@@ -1,248 +1,91 @@
-  --saltButton-background-active: var(--pill-background-disabled);
-  --saltButton-background-disabled: initial;
-}
-
-/* Style applied to Pill delete button icon  */
-.saltPill-deleteButton .saltPill-deleteIcon {
-  left: var(--saltPill-deleteButton-left, var(--pill-icon-offset));
-  position: relative;
-}
-
-/* Style applied to Pill delete button icon on active state  */
-.saltPill-deleteButton:active .saltPill-deleteIcon {
-  --saltIcon-color: var(--saltPill-deleteButton-color-active, var(--pill-icon-color-active));
-}
-
-/* Style applied to Pill delete button icon if `disabled={true}` on active state  */
-.saltPill-disabled:active .saltPill-deleteIcon {
-  --saltButton-background-active: var(--pill-background-disabled);
-  --saltIcon-color: var(--saltPill-deleteButton-color-disabled, var(--pill-icon-color));
-}
-
-/* Style applied to Pill icon */
-.saltPill .saltPill-icon {
-  fill: var(--pill-icon-color);
-  height: 12px; /* TODO: Replace with updated size */
-  margin-left: var(--salt-size-adornmentGap);
-  margin-right: calc(var(--salt-size-unit) * 0.5);
-}
-
-/* Style applied to Pill icon on active state */
-.saltPill:active .saltPill-icon,
-.saltPill-active .saltPill-icon {
-  fill: var(--pill-icon-color-active);
+/* Style applied to Pill if `onClick` prop is provided  */
+.saltPill-clickable {
+  cursor: var(--salt-selectable-cursor-hover);
+  user-select: none;
 }
 
-.saltPill:not(.saltPill-deletable) .saltPill-icon + .saltPill-label {
-  padding-left: 0;
+.saltPill-clickable:hover {
+  color: var(--pill-text-color-hover);
+  background: var(--pill-background-hover);
 }
 
-/* Pill label styles on hover */
-.saltPill:hover .saltPill-label {
-  color: var(--pill-text-color-hover);
+.saltPill-clickable[aria-disabled]:hover,
+.saltPill-nestedHover:hover {
+  color: var(--pill-text-color);
+  background: var(--pill-background);
 }
 
-/* Pill label styles on active state */
-.saltPill:active .saltPill-label {
+.saltPill-clickable.saltPill-active,
+.saltPill-clickable:active {
+  background: var(--pill-background-active);
   color: var(--pill-text-color-active);
 }
 
-/* Pill label styles when `selectable={true}` or `deletable={true}` */
-.saltPill-selectable .saltPill-label,
-.saltPill-selectable:active .saltPill-label,
-.saltPill-deletable .saltPill-label,
-.saltPill-deletable:active .saltPill-label {
+.saltPill-clickable[aria-disabled].saltPill-active,
+.saltPill-clickable[aria-disabled]:active,
+.saltPill-clickable.saltPill-nestedHover.saltPill-active,
+.saltPill-clickable.saltPill-nestedHover:active {
   background: var(--pill-background);
   color: var(--pill-text-color);
 }
 
-/* Pill label styles when  `disabled={true}` */
-.saltPill-disabled .saltPill-label,
-.saltPill-disabled:active .saltPill-label {
-  background: initial;
+/* Style applied to Pill on focus */
+.saltPill:focus-visible {
+  outline: var(--salt-focused-outline);
+}
+
+/* Style applied to Pill when disabled */
+.saltPill[aria-disabled],
```

Comment:
this differs from how we append the -disabled suffix to the root classname in other components

Reply:
Good point 👍